test/Microsoft.Health.Fhir.Stu3.Tests.E2E/KnownCrucibleTests.cs

```diff
@@ -181,6 +181,7 @@ public class KnownCrucibleTests
             "search001/SE05.5P",
             "search001/SE05.6G",
             "search001/SE05.6P",
+            "search001/SE06P",
```

Comment:
how come this fails?

Reply:
Because it is run against Cosmos which doesn't support `_include`. Previously it appears that it would execute and ignore, but it is now returned as an exception.

Reply:
Oh right.